core/src/main/java/org/pentaho/platform/workitem/WorkItemLifecyclePhase.java

```diff
@@ -17,72 +17,72 @@
 
 package org.pentaho.platform.workitem;
 
-import org.apache.commons.lang.builder.ToStringBuilder;
-import org.pentaho.platform.util.messages.Messages;
-
 /**
  * An enumeration of the known lifecycle events for the work item.
  */
 public enum WorkItemLifecyclePhase {
   /**
    * The work item has been submitted for execution
    */
-  SUBMITTED( "LifecyclePhase.SUBMITTED" ),
+  SUBMITTED,
   /**
    * The work item has been dispatched to the component responsible for its execution
    */
-  DISPATCHED( "LifecyclePhase.DISPATCHED" ),
+  DISPATCHED,
   /**
    * The work item has been received by the component responsible for its execution
    */
-  RECEIVED( "LifecyclePhase.RECEIVED" ),
+  RECEIVED,
   /**
    * The work item execution has been rejected
    */
-  REJECTED( "LifecyclePhase.REJECTED" ),
+  REJECTED,
   /**
    * The work item execution is in progress
    */
-  IN_PROGRESS( "LifecyclePhase.IN_PROGRESS" ),
+  IN_PROGRESS,
   /**
    * The work item execution has succeeded
    */
-  SUCCEEDED( "LifecyclePhase.SUCCEEDED" ),
+  SUCCEEDED,
   /**
    * The work item execution has failed
    */
-  FAILED( "LifecyclePhase.FAILED" ),
+  FAILED,
   /**
    * The work item execution has been restarted
    */
-  RESTARTED( "LifecyclePhase.RESTARTED" );
-
-  private String nameMessageKey;
-  private String name;
-  private String description;
-
-  WorkItemLifecyclePhase( final String nameMessageKey ) {
-    this.nameMessageKey = nameMessageKey;
-    this.name = getMessageBundle().getString( nameMessageKey );
-    this.description = getMessageBundle().getString( nameMessageKey + "_DESC" );
-  }
-
-  public String getName() {
-    return this.name;
-  }
-
-  public String getDescription() {
-    return this.description;
-  }
+  RESTARTED,
+  /**
+   * The work item execution has been restarted
+   */
+  REDIRECTED;
 
-  public String toString() {
-    return new ToStringBuilder( this )
-      .append( "name", this.getName() )
-      .append( "description", this.getDescription() )
-      .toString();
+  /**
+   * A convenience method that pads the string representation of this {@link WorkItemLifecyclePhase} such that all
+   * padded names are of the same length. This allows us to line up values within logs and makes the logs slightly
+   * more readable.
+   *
+   * @return {@link String} containing a padded string representation of this {@link WorkItemLifecyclePhase}
+   */
+  public String getPaddedName() {
```

Comment:
Not sure this belongs in this class -- seems specific to logging / how the value is to be rendered..